src/sinks/humio/metrics.rs

```diff
@@ -102,6 +102,7 @@ impl SinkConfig for HumioMetricsConfig {
             tls: self.tls.clone(),
             timestamp_nanos_key: None,
             acknowledgements: Default::default(),
+            timestamp_key: "timestamp".to_string(), // hard coded as humio expects this format so no sense in making it configurable
```

Comment:
Question for other reviewers: I _feel_ like these comments are normally on the line above, rather than after the code on the same line. Is there a best practice/common usage here?

Reply:
Might just be in my head 😅

Reply:
I usually vary it depending on the length of the line. In this case, I'd probably put it above since it is a pretty long line.